### ISSUE TITLE
Multiple Lot/Serials reception when receiving supplier orders

### ISSUE THREAD

**Issue by minigitmax** [Dolibarr/dolibarr#18229]
# Feature Request
When receiving supplier orders with many serialized products, it could be very useful to enter the list of serials at once, without having to enter the single serial numbers line by line.

## Suggested implementation
Keep the current procedure but allow an alternative procedure in case of many serials to be received. The alternative procedure could be a text dialogue window where enter the serials list, e.g. separated by colon or semicolon. The current procedure is more than annoying when many serial numbers need to be received.

**Comment by minigitmax** [comment]
The issue is a feature request and the request is still relevant and not yet covered by latest stable version. Please consider the suggested implementation in the issue description.

**Comment by atm-maxime** [comment]
Agreed. Have a look on the inventory module. There's a new bulk import feature that should be adapted to work on orders receptions and shipping also.

**Comment by minigitmax** [comment]
The request is still relevant and not yet covered by latest stable version. Please keep it live and do your best to add this inprovement asap.

**Comment by minigitmax** [comment]
The request is still relevant and not yet covered by latest stable version. Please keep it live and try solve it asap. Thank you
